soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -165,7 +165,16 @@ endchoice
 
 choice SOC_HFXO_LOAD_CAPACITANCE
 	prompt "HFXO load capacitance"
-	default SOC_HFXO_CAP_EXTERNAL
+	default SOC_HFXO_CAP_DEFAULT
+
+config SOC_HFXO_CAP_DEFAULT
+	bool "Use default SoC settings"
```

Comment:
Maybe just: `SoC default`.

Currently:
![image](https://user-images.githubusercontent.com/9802583/122221138-49587800-ceb1-11eb-8f8f-c616e3f6e21a.png)

And proposed:
![image](https://user-images.githubusercontent.com/9802583/122221366-802e8e00-ceb1-11eb-9a55-0d2519329e0d.png)

Reply:
looking at the screenshots, the setting above could also benefit an update 😆 

```
LFXO load capacitance (7 pF internal load capacitance)
```
double "load capacitance".
but that is of course outside scope of this PR, just wanted to mention.
```
LFXO load capacitance (7 pF internal)
```